adapters/adnuntius/adnuntius.go

```diff
@@ -122,7 +122,7 @@ func makeEndpointUrl(ortbRequest openrtb2.BidRequest, a *adapter) (string, []err
 		q.Set("consentString", consent)
 	}
 
-	if deviceExt.NoCookies {
+	if deviceExt.NoCookies || noCookies {
 		q.Set("noCookies", "true")
```

Comment:
Generally the idea was that you could set `noCookies` it in both places, but maybe I should just get rid of the `deviceExt` part. No one is using it as of now. will change that. Just a general question since I'm pretty new to golang, is unmarshaling crazy expensive or just a bit expensive?

Reply:
Unmarshal library we are currently using is very robust and reliable, but expensive. `extDeviceAdnuntius` is not a huge struct so I don't think it'd be crazy expensive, but if `noCookies` is already `true`, there seems to be no reason to unmarshal `deviceExt`. I think it doesn't hurt to set `noCookies` in either place. For backwards compatibility I think it makes sense to leave `deviceExt`. 

I just think that if `noCookies` is `true`, the OR in line 125 would evaluate to `true` regardless of the value of `deviceExt.NoCookies`. Therefore, I think it makes sense to unmarshal `deviceExt` only if needed. That is, only if `noCookies` is `false`. I leave it up to you.